sdk/identity/identity/CHANGELOG.md

```diff
@@ -1,15 +1,11 @@
 # Release History
 
-## 4.1.1 (Unreleased)
+## 4.2.0 (2024-05-07)
```

Comment:
do we want to change it to an earlier date so that beta can be out on this date? Maybe May 1st?

Reply:
Yup, updated to 4/30 to release on Tuesday. Thanks!